### ISSUE TITLE
Nothing happens when clicking "accept" for accepting a project application

### ISSUE THREAD

**Issue by SunyataZero** [Code4SocialGood/c4sg-web#1829]
Steps to reproduce:
1. Login as a organizational user
2. Go to a project page that you have created, and where there is a user that has sent in an application which you have not accepted yet
3. Click on the "accept" button
4. Result: There is a "click response" for the button, but other than that nothing happens, in other words the checkmark doesn't change to a double checkmark like expected

Please note:
* I've tried on two different projects
* This wasn't a problem before
* I've tried on both Firefox and Chrome (latest versions, and i'm running Lubuntu 17.10)

Console output:
* [console-cfsg-firefox.txt](https://github.com/Code4SocialGood/c4sg-web/files/1943415/console-cfsg-firefox.txt)
* [console-cfsg-chrome.txt](https://github.com/Code4SocialGood/c4sg-web/files/1943416/console-cfsg-chrome.txt)

**Comment by zhufeilong** [comment]
spot the problem caused by the followed codes in auth.service.ts, but I don't know how to get the right access_token when using email login.

`         // email signups do not readily return a usable JWT, so we need to grab
          // the temporary token. social sign ups, however, do return the jwt, so we
          // don't do anything special in this case.

          try {
              jwtHelper.decodeToken(authResult.accessToken);
          } catch (e) {
              localStorage.setItem('access_token', authResult.idTokenPayload['http://tempToken']);
          }`

**Comment by felipealves-gnu** [comment]
@SunyataZero which URL did you try ? That happens on DEV or PROD environment ?

**Comment by SunyataZero** [comment]
@felipealves-gnu Hi, it was in PROD (the live website). Not sure about the URL, haven't kept that data from when i tested, do you not get the same problem if following the steps to reproduce the problem? Kind Regards, Tord

**Comment by SunyataZero** [comment]
Hi i still have this problem, i get "EXCEPTION: JWT must have 3 parts" on firefox on my Lubuntu system. (Haven't looked at other browsers/systems)

**Comment by felipealves-gnu** [comment]
@SunyataZero can you send a print screen with this error happening on your browser, please ?

**Comment by SunyataZero** [comment]
@felipealves-gnu Well it doesn't show anything, the only thing that happens is that the application is not accepted:

![image](https://user-images.githubusercontent.com/10245688/41848760-e0a6f59c-787e-11e8-870c-43ea06541881.png)

So nothing really happens except for a small click response on the button

Do you want me to try using chrome or another browser? Something else i can do?

**Comment by felipealves-gnu** [comment]
@SunyataZero yes try with chrome. But before click on the button please go to 3 on the upper right side (on chrome) -> More tools -> Developer tools. You will see a bar on the bottom, so then you choose NETWORK (and check an option called "Preserve log") check the requests made after the click action. So copy and  make a screen shot, please

**Comment by SunyataZero** [comment]
@felipealves-gnu I'm having the same problem on Chrome. Just installed it so there cannot be a chache problem (new OS, new installation of Chrome). Have you tested this yourself and it works for you?

Here's the logs for Chrome:

At reload of webpage: [at-reload_app.code4socialgood.org-1530358201733.log](https://github.com/Code4SocialGood/c4sg-web/files/2151659/at-reload_app.code4socialgood.org-1530358201733.log)

At accept of application: [at-accept_app.code4socialgood.org-1530358241213.log](https://github.com/Code4SocialGood/c4sg-web/files/2151660/at-accept_app.code4socialgood.org-1530358241213.log)

(Screenshot is same as above, in other words nothing happens except the graphical click response)

Btw my OS is Lubuntu 18.04 64-bit (more or less the same as Ubuntu 18.04 64-bit)

**Comment by SunyataZero** [comment]
Now when i try to accept an application it gives me an error message in the upper right corner, but after i reload the page the application has been accepted

I have looked at [another issue](https://github.com/Code4SocialGood/c4sg-web/issues/1832) and through that found that we need to update the auth0 and lock js lib versions. When i check the source i can see this:
```
  <script src="https://cdn.auth0.com/js/lock/10.8/lock.min.js"></script>
  <script src="https://cdn.auth0.com/js/auth0/8.8/auth0.min.js"></script>
```
But we need to upgrade to a later version. **[See this announcement for more info](https://community.auth0.com/t/upgrade-reminder-changes-to-deprecation-roadmap/9552)**

In short, if things are like they were in the other issue we may only need to upgrade the 10.8 and 8.8 to a newer version
